apisix/plugins/example-plugin.lua

```diff
@@ -85,6 +86,27 @@ function _M.access(conf, ctx)
     core.log.warn("plugin access phase, conf: ", core.json.encode(conf))
     -- return 200, {message = "hit example plugin"}
 
+    if conf.server_func then
+        -- Example plugin server function call
+        local res, err = core.server_func(conf.server_func, core.json.encode({
+            ["data"] = "body-content"
+        }), {
+            ["Content-Type"] = "application/json",
+        })
+        if not err then
+            core.log.warn("plugin server func result: ", res.body)
+            if res.status == 200 then
+                core.response.set_header("Server-Func-Response", res.body)
```

Comment:
return the response body?

Reply:
User may need other information, such as res.status